literature.bib

```diff
@@ -372,7 +372,29 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FK:POS24,
+	title       = {{Life Span of SAT Techniques}},
+	author      = {Mathias Fleury and Daniela Kaufmann},
+	booktitle   = POS,
+	institution = arXiv,
+	doi         = {10.48550/arXiv.2402.01202},
+	year        = 2024,
+	ek-tags     = {satisfiability, preprocessing}
+}
 
+@article{YOL+:FSE24,
```

Comment:
I have agreed with Thomas to treat the entries of FSE as they are; i.e., they are published in the new PACMSE journal. Nonetheless, to track the original origin, we keep FSE in the key of the entry.